onnxruntime/python/tools/kernel_explorer/README.md

```diff
@@ -1,12 +1,37 @@
-# Kernel Explorer 
+# Kernel Explorer
+
 Kernel Explorer hooks up GPU kernel code with a Python frontend to help develop, test, profile, and auto-tune GPU kernels.
 
-## Example Usage
-```
-mkdir build
-cd build
-cmake -D onnxruntime_ROCM_HOME=/opt/rocm ..
-cmake --build .
-pytest ../kernels --verbose
-python ../kernels/vector_add_test.py
+Kernel explorer mainly target for attention based models and the purpose is specific to it at the moment.
```

Comment:
Maybe we don't need this line. If we want to add something, maybe something like this: The initial scope is for BERT-like models with ROCM EP, but the infrastructure is quite general and could potentially be extended for other models (ops) and EPs.